client/blocks/term-tree-selector/terms.jsx

```diff
@@ -388,6 +388,17 @@ const TermTreeSelectorList = React.createClass( {
 		);
 	},
 
+	cellRendererWrapper: function( { key, style, ...rest } ) {
+		return (
+			<div
+				className="Grid__cell"
```

Comment:
Unless I've missed it, I don't think there are any styles associated with `Grid__cell` or that are otherwise needed for the wrapper. Should be safe to drop `className` altogether, yeah?